Makefile.am

```diff
@@ -41,6 +41,7 @@ include tools/tools.mk
 include doc/doc.mk
 
 install-data-hook:
+	getent group pkcs11 > /dev/null || $(GROUPADD) -r pkcs11
 if ENABLE_LIBRARY
```

Comment:
Do we need to mention this changed behavior in the readme or manpage? Or even the opencryptoki book?

Reply:
This is only during 'make install'. Without that it hits you when you build your own opencryptoki from source and want to install it without previously having the distro-opencryptoki installed (which did create the pkcs11 group), or created the group manually before. 

So this change will not make any difference for distro installed opencryptoki.